lms/djangoapps/instructor_analytics/tests/test_basic.py

```diff
@@ -455,7 +455,11 @@ def test_sale_order_features_with_discount(self):
         order.purchase()
 
         # get the updated item
-        item = order.orderitem_set.all().select_subclasses()[0]
+        import pdb; pdb.set_trace()
```

Comment:
Still a pdb reference here.

Reply:
Nah - I accidentally push a WIP file from another ticket I'm working on.  I did catch it - but apparently not as fast as you did.  ;)